lang/LangSource/PyrMathOps.cpp

```diff
@@ -648,10 +648,10 @@ int doSpecialBinaryArithMsg(VMGlobals* g, int numArgsPushed, bool isPrimitive) {
                 SetRaw(a, slotRawInt(a) ^ slotRawInt(b));
                 break;
             case opLCM:
-                SetRaw(a, sc_lcm((long)slotRawInt(a), (long)slotRawInt(b)));
+                SetRaw(a, static_cast<int32_t>(sc_lcm((long)slotRawInt(a), (long)slotRawInt(b))));
```

Comment:
Instead, would it make sense to overload `sc_lcm` so that these combinations of types behave correctly (e.g. move casts to one place)?

Reply:
```
inline long sc_lcm(long a, long b) 
inline long sc_lcm(int a, int b) 
```
On windows long is the same size as int, that is way the cast must go here.
I think sc should be moved to fixed width types throughout, but that is a different PR.